libreosteoweb/models.py

```diff
@@ -418,10 +431,6 @@ class OfficeSettings(models.Model):
                                               blank=True)
 
     def save(self, *args, **kwargs):
-        """

```

Comment:
cleanup : overriden save method can be removed.

Reply:
done... a mistake